youtube_dl_gui/locale/cs_CZ/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,589 @@
+msgid "Info"
+msgstr "Informace"
+
+#: youtube_dl_gui/mainframe.py:102
+msgid "Welcome"
+msgstr "Vítejte"
+
+#: youtube_dl_gui/mainframe.py:103
+msgid "Warning"
+msgstr "Upozornění"
+
+#: youtube_dl_gui/mainframe.py:105
+msgid "Add"
+msgstr "Přidat"
+
+#: youtube_dl_gui/mainframe.py:106
+msgid "Download list"
+msgstr "Seznam stahovaných"
+
+#: youtube_dl_gui/mainframe.py:107 youtube_dl_gui/mainframe.py:516
+#: youtube_dl_gui/mainframe.py:534
+msgid "Delete"
+msgstr "Smazat"
+
+#: youtube_dl_gui/mainframe.py:108
+msgid "Play"
+msgstr "Přehrát"
+
+#: youtube_dl_gui/mainframe.py:109
+msgid "Up"
+msgstr "Nahoru"
+
+#: youtube_dl_gui/mainframe.py:110
+msgid "Down"
+msgstr "Dolů"
+
+#: youtube_dl_gui/mainframe.py:111
+msgid "Reload"
+msgstr "Obnovit"
+
+#: youtube_dl_gui/mainframe.py:112 youtube_dl_gui/mainframe.py:448
+#: youtube_dl_gui/mainframe.py:649
+msgid "Pause"
+msgstr "Pozastavit"
+
+#: youtube_dl_gui/mainframe.py:113 youtube_dl_gui/mainframe.py:865
+#: youtube_dl_gui/mainframe.py:866 youtube_dl_gui/optionsframe.py:587
+msgid "Start"
+msgstr "Spustit"
+
+#: youtube_dl_gui/mainframe.py:114
+msgid "About"
+msgstr "O aplikaci"
+
+#: youtube_dl_gui/mainframe.py:115
+msgid "View Log"
+msgstr "Zobrazit záznam"
+
+#: youtube_dl_gui/mainframe.py:117
+msgid "Successfully downloaded {0} URL(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "Úspěšně staženo {0} URL za {1} dní {2} hodin {3} minut {4} sekund"
+
+#: youtube_dl_gui/mainframe.py:119
+msgid "Downloads completed"
+msgstr "Stahování dokončeno"
+
+#: youtube_dl_gui/mainframe.py:120
+msgid "Total Progress: {0:.1f}% | Queued ({1}) Paused ({2}) Active ({3}) Completed ({4}) Error ({5})"
+msgstr "Celkový postup: {0:.1f}% | Ve frontě ({1}) Pozastaveno ({2}) Aktivní ({3}) Dokončené ({4}) Chyby ({5})"
+
+#: youtube_dl_gui/mainframe.py:121
+msgid "Stopping downloads"
+msgstr "Stahování se zastavuje"
+
+#: youtube_dl_gui/mainframe.py:122
+msgid "Downloads stopped"
+msgstr "Stahování zastaveno"
+
+#: youtube_dl_gui/mainframe.py:123
+msgid "You need to provide at least one URL"
+msgstr "Je třeba zadat alespoň jednu URL adresu"
+
+#: youtube_dl_gui/mainframe.py:124
+msgid "Downloads started"
+msgstr "Stahování zahájeno"
+
+#: youtube_dl_gui/mainframe.py:125
+msgid "Choose Directory"
+msgstr "Vyberte adresář"
+
+#: youtube_dl_gui/mainframe.py:127
+msgid "Download in progress. Please wait for all downloads to complete"
+msgstr "Stahuje se. Počkejte prosím, až se dokončí všechna stahování"
+
+#: youtube_dl_gui/mainframe.py:128
+msgid "Update already in progress"
+msgstr "Aktualizace již probíhá"
+
+#: youtube_dl_gui/mainframe.py:130
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "Stahuje se nejnovější youtube-dl. Počkejte prosím..."
+
+#: youtube_dl_gui/mainframe.py:131
+msgid "Youtube-dl download failed [{0}]"
+msgstr "Stahování youtube-dl selhalo [{0}]"
+
+#: youtube_dl_gui/mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "youtube-dl byl úspěšně stažen"
```

Comment:
You should change *youtube-dl* to ***Y**outube-dl*.

Reply:
ok